chandra_aca/maude_decom.py

```diff
@@ -1191,11 +1191,14 @@ def _get_aca_packets(
     table["IMGCOL0"][table["IMGTYPE"] == 2] -= 1
 
     table["INTEG"] = table["INTEG"] * 0.016
-    table["END_INTEG_TIME"] = table["TIME"] + table["INTEG"]
+    table["END_INTEG_TIME"] = table["TIME"] - 1.025
```

Comment:
I won't really argue with this too much, but it seems very odd to me that the image `TIME` is 1.025 sec **after** the integration ends (`END_INTEG_TIME`), for the default case of `adjust_time=False`. I can't picture a use case for combined images where this definition of `TIME` is useful.

In any case, given all our confusion surrounding these two time tags, this PR should add clear documentation on the definition of `TIME` and `END_INTEG_TIME`.

Reply:
> I won't really argue with this too much, but it seems very odd to me that the image `TIME` is 1.025 sec **after** the integration ends (`END_INTEG_TIME`), for the default case of `adjust_time=False`

This was one of my "aha" moments during our discussion yesterday, and I remember I asked "so the END_INTEG_TIME is before TIME?" (I'm guessing you missed my question, because we did not discuss it). The point is that the OBC must assign the time 1.025 seconds after the integration ended. That makes sense to me, and it also agrees with the timeline you were drawing on the board.